<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widgetMeasure</class>
 <widget class="QWidget" name="widgetMeasure">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>868</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Свипирование по мощности</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,4">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="lblParam">
       <property name="text">
        <string>F=</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QDoubleSpinBox" name="spinParam">
       <property name="suffix">
        <string> кГц</string>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="lblSecMin">
       <property name="text">
        <string>Pmin=</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spinSecMin"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="lblSecMax">
       <property name="text">
        <string>Pmax=</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spinSecMax"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblSecStep">
       <property name="text">
        <string>Pstep=</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="spinSecStep"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableView" name="tabResult">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>20</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>20</number>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
